Doc/docstrings_to_rst.py

```diff
@@ -76,6 +86,38 @@ def make_configurator_doc(conf: str, parent: str) -> str:
         result += f"- {iname}: :ref:`configurator-converter-{conf}` default={defval}\n"
     converter_page.append(result)
 
+
+for qvecs in generators:
+    result = ""
+    temp: IQVectors = IQVectors.create(qvecs, unit_cell=UnitCell(np.eye(3)))
+    if not temp.enabled:
+        continue
+    result += f"\n.. _qvectors-reference-{qvecs}:\n\n"
+    result += f"{qvecs}\n"
+    result += "".join(len(qvecs)*['~'])+'\n\n'
+    if temp.__doc__:
+        result += '\n'.join(str(x).lstrip() for x in temp.__doc__.split('\n'))
```

Comment:
Is [`textwrap.dedent`](https://docs.python.org/3/library/textwrap.html#textwrap.dedent) what you want here?

If not could be simpler (since `.split` implies it's a string anyway):

```suggestion
        result += "\n".join(map(str.lstrip, temp.__doc__.splitlines()))
```